enzy_htp/structure/residue.py

```diff
@@ -402,6 +401,18 @@ def shift(self, point:List[float] ) -> None:
 
     #endregion
 
+    def create_seq_res(self) -> chem.SeqRes:
+        """TODO(CJ): finish documentation"""
+
+        return chem.SeqRes(
+            model=1,
+            chain=self.parent.name,
+            idx=self.idx,
+            seq_idx=None,
+            name=self.name,
+            missing=False
+        )
+
```

Comment:
This can go to the === Getter-Prop === region and maybe rename as `get_seq_res()`

Reply:
sure. I renamed it to just `seqres()`, added the `@property` method and put it in that section.

Reply:
Cool!

Reply:
I moved it to the correct region btw